Makefile

```diff
@@ -99,6 +99,7 @@ build_runtime_image_tag=
 # examples repository
 examples_uri=
 examples_ref=
+examples_path=
```

Comment:
Can't we reuse the `examples_uri` and `examples_ref` and use Git from a golang library ?

Reply:
Will take a look on it.